src/python/pants/engine/internals/mapper_test.py

```diff
@@ -137,66 +106,3 @@ def test_address_family_duplicate_names() -> None:
                 AddressMap("name/space/1", {"one": Thing(name="one", age=37)}),
             ],
         )
-
-
-class HydratedStructs(Collection[HydratedStruct]):
-    pass
-
-
-@rule
-async def unhydrated_structs(addresses: Addresses) -> HydratedStructs:
-    tacs = await MultiGet(Get(HydratedStruct, Address, a) for a in addresses)
-    return HydratedStructs(tacs)
-
-
-class AddressMapperTest(unittest.TestCase, SchedulerTestBase):
```

Comment:
This was testing things that are beyond the scope of `mapper.py`. The functionality is roughly covered by the new tests in `build_files_test.py`.